common/source/docs/common-lightware-sf10-lidar.rst

```diff
@@ -13,169 +13,176 @@ distance measurements. The series includes a number of models:
 of the UART to the TX line of the Lidar, and the TX line of the UART to
 the RX line of the Lidar. Also connect the GND and 5V lines. You do not
 need flow control pins.
-
+
 The diagram below shows how to connect to SERIAL4.
-
-.. figure:: ../../../images/Pixhawk_Rangefinder_SF10_Serial.jpg
-   :target: ../_images/Pixhawk_Rangefinder_SF10_Serial.jpg
-
-   Pixhawk and SF10Rangefinder: Serial Connection (SERIAL4)
-
+
+.. figure:: ../../../images/Pixhawk_Rangefinder_SF10_Serial.jpg
+   :target: ../_images/Pixhawk_Rangefinder_SF10_Serial.jpg
+
+   Pixhawk and SF10Rangefinder: Serial Connection (SERIAL4)
+
 You then need to setup the serial port and rangefinder parameters. If
 you have used the SERIAL4/5 port on the Pixhawk then you would set the
 following parameters (this is done in the *Mission Planner*
 **Config/Tuning \| Full Parameter List** page):
-
+
 -  ``SERIAL4_PROTOCOL=9``
 -  ``SERIAL4_BAUD=19200``
 -  ``RNGFND_TYPE=8`` (LightWareSerial)
 -  ``RNGFND_SCALING=1``
 -  ``RNGFND_MIN_CM=5``
 -  ``RNGFND_MAX_CM=5000``
 -  ``RNGFND_GNDCLEAR=`` *Your offset to ground*
-
+
 The ``RNGFND_MAX_CM`` is the distance in centimeters that rangefinder
 can reliably read. The value of this setting depends on the model of
 Lidar (50 meters is the nominal maximum range for the
 `SF10/B <http://www.lightware.co.za/shop/en/shop/en/rangefinders-and-altimeters/32-sf10b.html>`__).
-
+
 If you instead were using the Telem2 port on the Pixhawk then you would
 use the ``SERIAL2_PROTOCOL`` and ``SERIAL2_BAUD`` parameters.
-
+
 ``RNGFND_GNDCLEAR`` is the distance in centimetres from the range finder
 to the ground when the vehicle is landed (the parameter value will
 depend on how you have mounted the rangefinder).
-
+
 I2C connection
 --------------
-
-.. warning::
-
+
+.. warning::
+
    I2C support for this rangefinder is not present in Copter 3.3
    or Rover 2.50 (planned for Copter 3.4). Support is present in Plane in
-   the current stable release (Plane 3.4).
-
+   the current stable release (Plane 3.4).
+
 Connect the SDA line of the Lidar to the SDA line of the I2C port on the
 Pixhawk, and the SCL line of the Lidar to the SCL line of the I2C port.
 Also connect the GND and 5V lines.
-
-.. figure:: ../../../images/Pixhawk_Rangefinder_SF10_I2C.jpg
-   :target: ../_images/Pixhawk_Rangefinder_SF10_I2C.jpg
-
-   Pixhawk and SF10 Rangefinder: I2CConnection
-
+
+.. figure:: ../../../images/Pixhawk_Rangefinder_SF10_I2C.jpg
+   :target: ../_images/Pixhawk_Rangefinder_SF10_I2C.jpg
+
+   Pixhawk and SF10 Rangefinder: I2CConnection
+
 You then need to configure the rangefinder parameters as shown below
 (this is done in the *Mission Planner* **Config/Tuning \| Full Parameter
 List** page):
-
+
 -  ``RNGFND_TYPE=7`` (LightWareI2C)
 -  ``RNGFND_SCALING=1``
 -  ``RNGFND_MIN_CM=5``
 -  ``RNGFND_MAX_CM=5000``
 -  ``RNGFND_GNDCLEAR=`` *Your offset to ground*
--  ``RNGFND_ADDR=85`` (I2C Address of lidar in decimal)
-
+-  ``RNGFND_ADDR=102`` (I2C Address of lidar in decimal)

```

Comment:
manual for SF10/C:
file:///C:/Users/tpittenger/Downloads/SF10%20-%20Laser%20Altimeter%20Manual%20-%20Rev%2010.pdf
I don't see any default address, just says it can be set by USB